doc/modules/mod_amp.md

```diff
@@ -1,13 +1,10 @@
 ### Module Description
 
 This module enables support for a subset of the functionality described under
-[XEP-0079: Advanced Message
-Processing](http://xmpp.org/extensions/xep-0079.html). It currently does not
-provide features related to timed delivery, i.e the `expire-at` condition.
-
-The `error` and `notify` actions are supported, while `alert` and `drop` are
-not. See more below, under XEP Support.
+[XEP-0079: Advanced Message Processing](http://xmpp.org/extensions/xep-0079.html). It currently does not
```

Comment:
[XEP-0079: Advanced Message Processing](http://xmpp.org/extensions/xep-0079.html). It currently does not
provide features related to a timed delivery, i.e. the `expire-at` condition.